sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaRulesDefinition.java

```diff
@@ -45,6 +48,12 @@
  */
 public class JavaRulesDefinition implements RulesDefinition {
 
+  private final boolean isDebugEnabled;
+
+  public JavaRulesDefinition(Settings settings) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)